src/main/java/com/github/dockerjava/api/model/PushResponseItem.java

```diff
@@ -18,7 +18,7 @@
      */
     @JsonIgnore
     public boolean isErrorIndicated() {
-        if (getError() == null)
+        if (getError() == null && getErrorDetail() == null)
```

Comment:
Such refactoring is exceeding the scope of this pull request, which is to get docker-java to stop rejecting valid ProgressDetail messages.